frontend/src/scenes/billing/BillingLimitInput.tsx

```diff
@@ -78,7 +78,7 @@ export const BillingLimitInput = ({ product }: { product: BillingProductV2Type }
         return null
     }
     return (
-        <div className="border-t border-border p-8">
+        <div className="border-t border-border p-8" data-attr={`billing-limit-input-${product.type.replace('_', '-')}`}>
```

Comment:
why do we do this here? `product_type` should return snake_case from the billing service, so it seems to make sense to keep it snake_case here.

Reply:
😅 I wanted the whole data-attr to be kebab case so it would look more consistent...I'll change it back.